editor/themes/editor_theme_manager.cpp

```diff
@@ -1849,9 +1849,14 @@ void EditorThemeManager::_populate_editor_styles(const Ref<EditorTheme> &p_theme
 
 			// When pressed, add a small bottom border to the buttons to better show their active state,
 			// similar to active tabs.
+			Ref<StyleBoxFlat> editor_log_button_normal = style_flat_button->duplicate();
+			Ref<StyleBoxFlat> editor_log_button_hover = style_flat_button_hover->duplicate();
```

Comment:
You don't need to duplicate them if no additional changes are made compared to these two styleboxes. Slight waste of resources, quite literally.

Reply:
If you are not modifying them, you don't even need the new variables.